tox.ini

```diff
@@ -156,8 +150,40 @@ commands =
     wheels: python setup.py sdist --formats=gztar,zip --dist-dir={toxinidir}/wheelhouse
     wheels: ls -l {toxinidir}/wheelhouse
 
+
+#
+# `apidocs` environment is designed to build only the API doc HTML files.
+#
+# API docs build errors are visible to stdout
+# The resulting files can be viewed with web browsers from the local `apidocs`
+# path.
+#
+# This is only used for development and manual and automated testing.
+#
 [testenv:apidocs]
-deps=https://github.com/twisted/pydoctor/archive/3f9c64829dfa040b334c9ae27c332c7078356e79.zip
+extras = docs
+commands = {toxinidir}/bin/admin/build-apidocs {toxinidir}/src/ apidocs
+
+
+#
+# `narrativedocs` environment is designed to build the non-API doc HTML files.
+#
+# To see all warning in a single run, this is not called with -W.
+# Instead, all logs are placed into a file.
+# This file triggers a failure when not empty.
+#
+# This is only used for development and manual and automated testing.
+#
+[testenv:narrativedocs]
+extras = docs
+setenv =
+    # Set this to `True` to run similar to Read the docs.
+    READTHEDOCS=False
+    READTHEDOCS_LANGUAGE=en
+    READTHEDOCS_VERSION=1.0.0.dev0
+commands = sphinx-build -aW -b html -d {toxinidir}/docs/_build {toxinidir}/docs {toxinidir}/docs/_build/
+#{toxinidir}/bin/admin/build-docs {toxinidir}
```

Comment:
this is here as a reminder for reviewers to see if we will still need bin/admin/build-docs or it can be removed.... or it should be updated so that tox will call bin/admin/build-docs instead of the custom sphinx-build